### ISSUE TITLE
Use Github Pages and a commit hook to extract user info with the Github API

### ISSUE THREAD

**Issue by aschrijver** [Widdershin/programmers-oath#126]
An interesting extension to this repository would be to have a published webpage (using Github Pages) and a commit hook, which kicks in when a new entry to SIGNATURES is added, and then uses the Github API to extract the display name of the user and maybe her/his avatar picture. After which it then automatically republishes the updated webpage.

**Comment by vassudanagunta** [comment]
I think that would be a violation of trust to do something like that without advanced consent from the signers. The signers came here to sign an oath. They didn't give permission to be part of the marketing of it.  

Given what this Oath is about, such trespass would look really bad too.

**Comment by aschrijver** [comment]
I tend to disagree. You sign with your GH name currently. IMHO that name stands for you, so should also be willing to adhere your GH display name with it. Whether your name is in the SIGNATURES file, or in in a HTML page should not be a big difference. Of course people should be made aware of the page.

Also this is not marketing, but yes, it is giving more awareness to those who signed the oath. Currently the SIGNATURES page with just a GH username is quite anonymous.
